tutorial/mfxx49820_bayesian_opt.yaml

```diff
@@ -0,0 +1,98 @@
+setup:
```

Comment:
Can we move the new task parameter defintions to the file located in the `config` directory. It is more up to date and uses paths consistent with running on S3DF.